app/assets/javascripts/admin/addon/components/site-settings/file-types-list.hbs

```diff
@@ -0,0 +1,32 @@
+<ListSetting
+  @value={{this.settingValue}}
+  @settingName={{this.setting.setting}}
+  @choices={{this.settingChoices}}
+  @onChange={{this.onChangeListSetting}}
+  @onChangeChoices={{this.onChangeChoices}}
+  @options={{hash allowAny=this.allowAny}}
+/>
+
+<DButton
+  @action={{(fn this.insertDefaultTypes "image")}}
+  @label="admin.site_settings.file_types_list.add_image_types"
+  class="btn file-types-list__button"
```

Comment:
I think these classes could also be a little more specific to allow for more targeting in specs; e.g. `file-types-list__button -image-types`, `file-types-list__button -video-types`

Reply:
Another improvement for all these buttons would be to add a `@title` attribute that says something like "Allow extensions `VIDEO_TYPES.join(",")` so you can see what will be added here by hovering over the button.